atlp-rwanda/e-commerce-mavericcks-bn
# Database configurations sequelize-postgres

## Purpose

The purpose of this pull request is to configure the database settings in the project.

## Changes Made

- Added configuration settings for the database in the project.
- Updated the database connection code to use the new configuration.
- Ensured that the database configuration is securely stored and accessed.

## Testing Instructions

To test the database configuration changes:

## Testing Instructions

To test the database configuration changes:

1. Clone the repository locally.
2. Install dependencies using `npm install`.
3. Set up the database according to the provided configuration settings. Ensure that you have a DBMS preferably PostgreSQL installed. if not  [download it here](https://www.postgresql.org/download/)
4. Run the migration script to create the necessary tables using `npm run migrate`.
5. Run the project using `npm run dev`.
6. Verify that the project connects to the database without errors.
7. Use Postman to test the user endpoints (localhost:8080/users/) and ensure proper functionality.

Additional Steps:
- Ensure that the `npm run migrate` command executes successfully and creates the required tables in the database.
- Test the user endpoints using Postman to ensure that the user is created successfully.
- Verify that the database is populated with the expected data after running the migration and testing the endpoints.


## Related Issues

None

## Checklist
